pkg/query/flamegraph_arrow.go

```diff
@@ -622,3 +699,10 @@ keys:
 
 	return intersection
 }
+
+func bytesToString(b []byte) string {
```

Comment:
I would make sure it's clear from the caller's perspective that this is unsafe. I like to call these `unsafeString` for example.